frontend/src/pages/Projects/Project/ProjectDetails/BudgetSection/FormConfig.ts

```diff
@@ -2,6 +2,14 @@ import { AxiosResponse } from "axios";
 import { UseQueryResult } from "@tanstack/react-query";
 import { IEditField } from "types";
 import { useParams } from "react-router-dom";
+import { object, string } from "yup";
+
+const validationSchema = object({
+  stob: string()
+  .min(4, 'Must contain exactly 4 alphanumeric characters.')
```

Comment:
the linter is complaining of a missing trailing comma, and some other formatting stuff

Reply:
Hmm, there was something up with my local linter somehow.

Reply:
Yeah, i just fixed the same thing - with the same npm package lol

not sure how that got removed